Wikipedia/Code/NotificationsCenterDetailView.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+
+final class NotificationsCenterDetailView: SetupView {
+
+    // MARK: - Properties
+
+    lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .insetGrouped)
+        tableView.register(NotificationsCenterDetailHeaderCell.self, forCellReuseIdentifier: NotificationsCenterDetailHeaderCell.reuseIdentifier)
+        tableView.register(NotificationsCenterDetailContentCell.self, forCellReuseIdentifier: NotificationsCenterDetailContentCell.reuseIdentifier)
+        tableView.register(NotificationsCenterDetailActionCell.self, forCellReuseIdentifier: NotificationsCenterDetailActionCell.reuseIdentifier)
+
+        tableView.estimatedRowHeight = 44
+        tableView.rowHeight = UITableView.automaticDimension
+        
+        return tableView
+    }()
+
+    // MARK: - SetupView
+
+    override func setup() {
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+
+        addSubview(tableView)
+        NSLayoutConstraint.activate([
+            tableView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            tableView.topAnchor.constraint(equalTo: topAnchor),
+            tableView.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ])
+    }
+
+}
+
+// MARK: - Themeable
+
+extension NotificationsCenterDetailView: Themeable {
+
+    func apply(theme: Theme) {
+        backgroundColor = theme.colors.baseBackground
+        tableView.backgroundColor = theme.colors.baseBackground
```

Comment:
Might want to `reloadData()` here or in the view controller so a system theme change on this screen will refresh the cells.

Reply:
Good catch - updated! Also added in extra commit that adjusts some font sizes per design request.